liota/dccs/iotcc.py

```diff
@@ -240,6 +245,13 @@ def _properties(self, msg_id, entity_type, entity_id, entity_name, timestamp, pr
             msg["body"]["property_data"].append({"propertyKey": key, "propertyValue": value})
         return msg
 
+    def _get_properties(self, msg_id, res_uuid):
+        return {
+            "transactionID": msg_id,
+            "type": "get_properties",
```

Comment:
Can you please change the type to get_properties_resquest to match to get_properties_response?

Reply:
yes